src/Fields.php

```diff
@@ -934,12 +932,15 @@ protected function wf_crm_get_fields($var = 'fields') {
 
       // Fetch custom fields
       $custom_types = wf_crm_custom_types_map_array();
-      $custom_fields = wf_crm_apivalues('CustomField', 'get', array(
-        'is_active' => 1,
-        'custom_group_id' => array('IN' => array_keys($custom_sets)),
-        'html_type' => array('IN' => array_keys($custom_types)),
-        'options' => array('sort' => 'weight'),
-      ));
+      $custom_fields = [];
+      if (count($custom_sets) > 0) {
+        $custom_fields = wf_crm_apivalues('CustomField', 'get', [
+          'is_active' => 1,
+          'custom_group_id' => ['IN' => array_keys($custom_sets)],
+          'html_type' => ['IN' => array_keys($custom_types)],
+          'options' => ['sort' => 'weight'],
+        ]);
+      }
       foreach ($custom_fields as $custom_field) {
```

Comment:
Ran into this in the dblog (this is a webform that has custom fields on it -> so custom_group_id was empty) -> 

`The CiviCRM "CustomField get" API returned the error: "invalid criteria for IN" when called by function "wf_crm_get_fields" on line 937 of Fields.php with parameters: "Array ( [is_active] => 1 [custom_group_id] => Array ( [IN] => Array ( ) ) [html_type] => Array ( [IN] => Array ( [0] => Select [1] => Multi-Select [2] => Radio [3] => CheckBox [4] => Text [5] => TextArea [6] => RichTextEditor [7] => Select Date [8] => Link [9] => Select Country [10] => Multi-Select Country [11] => Select State/Province [12] => Multi-Select State/Province [13] => Autocomplete-Select [14] => File ) ) [options] => Array ( [sort] => weight [limit] => 0 ) [check_permissions] => [version] => 3 ) "`

Fixed by counting to ensure we have some before we go and retrieve them.